### PR TITLE
HTML bug Fix

### PR DESCRIPTION
**Pull request by leonardoeng13**

### CODE REVIEW

**Comment by leonardoeng13** [comment]
Great job fixing the bugs and improving your work.
For next time I believe you can run linters in your machine to avoid rejections. Create some tests to developed that skill ;-).

File: tribute_page.html [hunk 1]
```diff
@@ -1,20 +1,22 @@
-<body id='main'>
-  <header>
-    <section id="title">
```
Comment (leonardoeng13) [inline tribute_page.html:3]: It's a good developer skill to create a base file to start your projects. With that the job will be more quickly and easy to start.

File: tribute_page.html [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
+  <body>
       <h1>Dr. Norman Borlaug</h1>
       <h2 id="tribute-info">The man who saved a billion lives</h2>
       <div id="img-div">
-        <img id = "image" class="center" class="responsive" src="https://www.pbs.org/wgbh/americanexperience/media/filer_public_thumbnails/filer_public/56/7c/567c2141-1572-4455-9078-3ddc8609e127/borlaug-article-1.jpg__1000x567_q85_crop_subsampling-2.jpg" id="image"/ alt="Dr. Norman Borlaug training biologists in Mexico">
-          <a href="https://www.pbs.org/wgbh/americanexperience/features/green-revolution-norman-borlaug-race-to-fight-global-hunger/" target="_blank"></a>
-        </img>
-      <div>
-       <p for="image" id="image-label"><em>
-        Borlaug smiling in wheat field. Image, 1960s. International Maize and Wheat Improvement Center (CIMMYT)</em></p></div>
```
Comment (leonardoeng13) [inline tribute_page.html:12]: We know sometimes is hard to find a good image and we have to use from other sources/sites. Remember to identify from who is the source and grant the images credits.

File: tribute_page.html [hunk 3]
```diff
@@ -44,11 +46,5 @@ <h2>Here's a time line of Dr. Borlaug's life:</h2>
       </article>
       <hr/>
     </div>
-  </main>
-  <footer>
-    <div id="footer-content">
-      <secion>If you have time, you should read more about this incredible human being on his <a href="https://www.pbs.org/wgbh/americanexperience/features/green-revolution-norman-borlaug-race-to-fight-global-hunger/" id="tribute-link" target="_blank">American Experience</a>.
-      </section>
-    </div>
-  </footer>
```
Comment (leonardoeng13) [inline tribute_page.html:53]: Great job removing unnecessary information from your source code. With time you'll develop more skills to create better and better websites.

**Review by leonardoeng13 — COMMENTED** [review]
Great job fixing the bugs and improving your work.
Learning objectives
Apply best practices in HTML code.
Use semantic HTML tags.
Use CSS selectors correctly.
Use CSS box model.
Perform a code review for yourself on a past project.

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @leonardoeng13 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**